cli/script/command-executor.ts

```diff
@@ -101,9 +101,36 @@ export var confirm = (): Promise<boolean> => {
 }
 
 function accessKeyAdd(command: cli.IAccessKeyAddCommand): Promise<void> {
-    return sdk.addAccessKey(command.description)
+    return sdk.addAccessKey(command.friendlyName, command.maxAge)
         .then((accessKey: AccessKey) => {
-            log("Successfully created a new access key" + (command.description ? (" \"" + command.description + "\"") : "") + ": " + accessKey.name);
+            log(`Successfully created a new access key "${command.friendlyName}": ${accessKey.name}\n(Expires: ${new Date(accessKey.expires).toString()})`);
+        });
+}
+
+function accessKeyEdit(command: cli.IAccessKeyEditCommand): Promise<void> {
+    var willEditFriendlyName = !!command.newFriendlyName && command.oldFriendlyName !== command.newFriendlyName;
+    var willEditMaxAge = command.maxAge !== null && command.maxAge !== undefined;
```

Comment:
i suppose. maybe this is some common idiom that i'm just not aware of, but i think it might be good to add a comment here then instead that explains that max age could be 0.